packages/transmute-framework/migrations/2_deploy_contracts.js

```diff
@@ -16,45 +16,58 @@ var RBAC = artifacts.require('./TransmuteFramework/RBAC.sol')
 
 var EventStoreLib = artifacts.require('./TransmuteFramework/EventStoreLib.sol')
 
-var UnsafeEventStore = artifacts.require('./TransmuteFramework/UnsafeEventStore/UnsafeEventStore.sol')
-var UnsafeEventStoreFactory = artifacts.require('./TransmuteFramework/UnsafeEventStore/UnsafeEventStoreFactory.sol')
+var UnsafeEventStore = artifacts.require('./TransmuteFramework/UnsafeEventStore.sol')
+var UnsafeEventStoreFactory = artifacts.require('./TransmuteFramework/UnsafeEventStoreFactory.sol')
 
-var RBACEventStore = artifacts.require('./TransmuteFramework/RBACEventStore/RBACEventStore.sol')
```

Comment:
These should still be deployed, and they should still be tested.

Reply:
They are - the pathing was just updated to reflect proper directory structure. How this was resolving before is a mystery to me.